### PR TITLE
Add CORS support

### PR DESCRIPTION
**Pull request by pastudan**
This would help me as I build a front-end app that uses this API. That said, I certainly understand if you don't want to open it up to a bunch of web clients.

### CODE REVIEW

**Comment by pastudan** [comment]
Hey @jakewmeyer, apologies in advance for my lack of ruby skills! Looking at travis it seems like the reason my tests are failing is due to env variables being stripped out. I'll also try to get it running locally tomorrow, but just to confirm, should I grab a separate mongo container and pass credentials into the app container?

File: app.rb [hunk 1]
```diff
@@ -7,13 +7,30 @@
 require 'bundler/setup'
 require 'sinatra'
 require 'sinatra/subdomain'
+require 'sinatra/cross_origin'
 require 'json'
 require 'mongo'
 
 # Uses the modular version of Sinatra
 class SpacexAPI < Sinatra::Base
   register Sinatra::Subdomain
```
Comment (jakewmeyer) [inline app.rb]: Add
```Ruby
register Sinatra::CrossOrigin
```

**Comment by jakewmeyer** [comment]
Currently looking at a solution for the Travis issue. It strips the env variables when the pull request is from an outside contributor. Writing separate tests as a workaround.

I ran the tests locally, and everything passes just fine. Also created a branch and the Travis build was successful 👍 

Only thing I would add is a line to register the extension with Sinatra, it needs this to pass. Add it right under the Sinatra::Subdomain registration.
```Ruby
register Sinatra::CrossOrigin
```
After that's added, I'm fine with merging 👍

**Comment by jakewmeyer** [comment]
If you want to fully test locally, I include the latest dump of the database on the [release](https://github.com/r-spacex/SpaceX-API/releases) page under the latest release.

**Comment by jakewmeyer** [comment]
Okay, I've added just a rudimentary test to check if the app starts up, as a solution for pull requests currently. If you pull the changes, add the registration, and push again, the travis build should pass now.

File: app.rb (hunk 1 above)
Comment (pastudan) [inline app.rb]: 8d067ca

**Comment by pastudan** [comment]
Awesome, added the subclass registration... tests passing now. Thanks!

**Pull request merged**